README.md

```diff
@@ -104,7 +104,7 @@ Note that the only argument `trackPersonWithInfo` takes is a dictionary represen
 
 ## Anonymous Tracking Notice
 
-By default, Klaviyo will begin tracking unidentified users in your app once the SDK is initialized. This means you will be able to track events from users in your app without any user information provided. When an email or other primary identifier is provided Klaviyo will merge the data from the anonymous user to a new identified user.
+By default, Klaviyo will begin tracking unidentified users in your app once the SDK is initialized. This means you will be able to track events from users in your app without any user information provided. When an email or other primary identifier is provided Klaviyo will merge the data from the anonymous user to a new identified user. Prior to version 1.7.0, the Klaviyo SDK used the [IDFV](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking. After that version the SDK will use a cached UUID that is generated when the SDK is initalized. The SDK will also detect if an IDFV was used previously as an identifier and will use that instead of generating a new UUID.
```

Comment:
I might tweak this section slightly. Changes in **bold**

Prior to version 1.7.0, the Klaviyo SDK used the **Apple identifier for vendor (IDFV)** (https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking. **Starting with version 1.7.0,** the SDK will use a cached UUID that is generated when the SDK is **initialized**. **For existing anonymous profiles using IDFV, the SDK will continue to use IDFV, instead of generating a new UUID.**

Reply:
thx!

Reply:
```suggestion
By default, Klaviyo will begin tracking unidentified users in your app once the SDK is initialized. This means you will be able to track events from users in your app without any user information provided. When an email or other primary identifier is provided Klaviyo will merge the data from the anonymous user to a new identified user. Prior to version 1.7.0, the Klaviyo SDK used the [Apple identifier for vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking. Starting with version 1.7.0, the SDK will use a cached UUID that is generated when the SDK is initialized. For existing anonymous profiles using IDFV, the SDK will continue to use IDFV, instead of generating a new UUID.
```